examples/bundle/Dockerfile

```diff
@@ -7,8 +7,8 @@ RUN wget https://github.com/ipfs/kubo/releases/download/v0.15.0/kubo_v0.15.0_lin
 RUN tar xvf kubo.tar.gz
 RUN mv kubo/ipfs /usr/bin/ipfs
 RUN mkdir -p /usr/lib/extension-release.d/
-RUN echo ID=kairos > /usr/lib/extension-release.d/extension-release.kubo
-RUN echo VERSION_ID=$VERSION >> /usr/lib/extension-release.d/extension-release.kubo
+RUN echo KAIROS_ID=kairos > /usr/lib/extension-release.d/extension-release.kubo
+RUN echo KAIROS_VERSION_ID=$VERSION >> /usr/lib/extension-release.d/extension-release.kubo
```

Comment:
I think it's other way around systemd-sysext will look into ID.

See docs: https://www.freedesktop.org/software/systemd/man/systemd-sysext.html

However, seems we can just put "_any":
```
A simple mechanism for version compatibility is enforced: a system extension image must carry a /usr/lib/extension-release.d/extension-release.$name file, which must match its image name, that is compared with the host os-release file: the contained ID= fields have to match unless "_any" is set for the extension. If the extension ID= is not "_any", the SYSEXT_LEVEL= field (if defined) has to match. If the latter is not defined, the VERSION_ID= field has to match instead. If the extension defines the ARCHITECTURE= field and the value is not "_any" it has to match the kernel's architecture reported by [uname(2)](https://man7.org/linux/man-pages/man2/uname.2.html) but the used architecture identifiers are the same as for ConditionArchitecture= described in [systemd.unit(5)](https://www.freedesktop.org/software/systemd/man/systemd.unit.html#). System extensions should not ship a /usr/lib/os-release file (as that would be merged into the host /usr/ tree, overriding the host OS version data, which is not desirable). The extension-release file follows the same format and semantics, and carries the same content, as the os-release file of the OS, but it describes the resources carried in the extension image.
```

Reply:
gotcha, thanks!

Reply:
even with `ID="_any"`, this one is still causing trouble  https://github.com/kairos-io/kairos/actions/runs/4732001891/jobs/8401102181#step:7:2560

Reply:
@mudler unfortunately the `_any` special value, was only introduced in systemd 252 https://github.com/systemd/systemd/blob/main/NEWS#L1116 while the leap box where we run tests against has only 249 atm. I'm going to switch this test to use tumbleweed which comes with 253 already and works properly.